### PR TITLE
Make substatus column wider DAH-580

### PR DESCRIPTION
**Pull request by akegan**
DAH-580

Definitely annoying to have this in the pattern lib, but doesn't seem worth it to move it right before we redo all the styling. Just trying to get it to look somewhat reasonable for now
image

### CODE REVIEW

File: public/toolkit/styles/molecules/_react-tables.scss [hunk 1]
```diff
@@ -459,7 +460,7 @@ $narrow-column-width: 120;
   }
 
   .td-offset-right {
```
Comment (james-wills-sf) [inline public/toolkit/styles/molecules/_react-tables.scss:462]: I'm a little confused at why this class should have the substatus width applied, seems like the class is more generic than just substatus?

**Review by james-wills-sf — COMMENTED** [review]
@akegan just one question

File: public/toolkit/styles/molecules/_react-tables.scss (hunk 1 above)
Comment (akegan) [reply public/toolkit/styles/molecules/_react-tables.scss:462]: 🤷 it's really not that generic, it's for the column to the right of the status column in the lease-up applications table. 
whole thing needs to be overhauled, idea was just that this is a quick fix before we redo the styling. 

Maybe it'd help if I just rename the class?

**Comment by akegan** [comment]
whoops never merged this, but not worth it any more with the new table styling pr

**Pull request closed without merging**